src/rgw/setup/rgw_service.py

```diff
@@ -35,7 +35,7 @@ def start(conf: MappedConf, config_file, log_file, motr_trace_dir, index: str =
             args = shlex.split(args)
             sys.stdout.flush()
             sys.stderr.flush()
-            os.execl(cmd, cmd, *args)
+            os.execl(cmd, cmd, *args)  # nosec
```

Comment:
this is suppressing the warning. what justification is there to do same ? @shriya-deshmukh

Reply:
We need  radosgw service to run as main process in container, So this execl is replacing rgw_setup  calling process with our radosgw process. which is not possible through other ways (os.system, subprocess.run subpeocess.Popen etc)
So we have kept os.execl and disable the codacy warning for this line.

Reply:
Ok i remember now. Also, wanted to capture reason here